src/main/java/wooteco/member/controller/config/LonginInterceptor.java

```diff
@@ -0,0 +1,22 @@
+package wooteco.member.controller.config;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.springframework.web.servlet.HandlerInterceptor;
+import wooteco.member.service.AuthService;
+
+
+public class LonginInterceptor implements HandlerInterceptor {
+    private final AuthService authService;
+
+    public LonginInterceptor(AuthService authService) {
+        this.authService = authService;
+    }
+
+    @Override
+    public boolean preHandle(HttpServletRequest request, HttpServletResponse response, Object handler) throws Exception {
+        String accessToken = request.getHeader("Authorization");
+        authService.validateToken(accessToken);
+        return HandlerInterceptor.super.preHandle(request, response, handler);
```

Comment:
```suggestion
        return true;
```
super.preHandle 를 확인해보면 위와 동일한 동작일 것 같아요. 그래도 super 를 호출하는 것이 좋을까요?

Reply:
앗 그렇군요! 타고 들어가서 상위 인터페이스 default 메소드를 확인해 보니, true를 반환하고 있네요.
바로 반영했습니다!! 감사해요!!